### PR TITLE
Add 4.72.0 changelog

### PR DESCRIPTION
**Pull request by rileykarson**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,40 @@
...
+FEATURES:
+* **New Resource:** `google_certificate_manager_public_ca_external_account_key` ([#14983](https://github.com/hashicorp/terraform-provider-google/pull/14983))
+
+IMPROVEMENTS:
+* compute: added `provisioned_throughput` field to `google_compute_disk` used by `hyperdisk-throughput` pd type ([#14985](https://github.com/hashicorp/terraform-provider-google/pull/14985))
+* container: added field `security_posture_config` to resource `google_container_cluster` ([#14999](https://github.com/hashicorp/terraform-provider-google/pull/14999))
+* logging: added support for `locked` to `google_logging_project_bucket_config` ([#14977](https://github.com/hashicorp/terraform-provider-google/pull/14977))
+
+BUG FIXES:
+* container: fixed incorrect code block in the description of `google_container_cluster` ([#14969](https://github.com/hashicorp/terraform-provider-google/pull/14969))
```
Comment (slevenick) [inline CHANGELOG.md:12]: Remove docs-only fix?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,40 @@
...
+* bigquery: fixed an issue where api default value for `edition` field of `google_bigquery_reservation` was not handled ([#14961](https://github.com/hashicorp/terraform-provider-google/pull/14961))
+* cloudfunction2: fixed permadiffs of some fields of `service_config` in `google_cloudfunctions2_function` resource ([#14975](https://github.com/hashicorp/terraform-provider-google/pull/14975))
+* compute: fixed an issue with setting project field to long form in `google_compute_forwarding_rule` and `google_compute_global_forwarding_rule` ([#14996](https://github.com/hashicorp/terraform-provider-google/pull/14996))
+* gkehub: fixed an issue with setting project field to long form in `google_gke_hub_feature` ([#14996](https://github.com/hashicorp/terraform-provider-google/pull/14996))
+
+## 4.71.0 (June 27, 2023)
```
Comment (slevenick) [inline CHANGELOG.md:18]: Did this not get updated after the previous release?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,40 @@
-## 4.71.0 (Unreleased)
+## 4.72.0 (Unreleased)
+
+FEATURES:
+* **New Resource:** `google_certificate_manager_public_ca_external_account_key` ([#14983](https://github.com/hashicorp/terraform-provider-google/pull/14983))
+
+IMPROVEMENTS:
+* compute: added `provisioned_throughput` field to `google_compute_disk` used by `hyperdisk-throughput` pd type ([#14985](https://github.com/hashicorp/terraform-provider-google/pull/14985))
+* container: added field `security_posture_config` to resource `google_container_cluster` ([#14999](https://github.com/hashicorp/terraform-provider-google/pull/14999))
```
Comment (slevenick) [inline CHANGELOG.md:8]: Do we want consistency between this and the above line? "added field x" vs "added x field"

**Review by slevenick — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (rileykarson) [reply CHANGELOG.md:18]: Yeah, I was late so it wasn't present in the release commit

File: CHANGELOG.md (hunk 3 above)
Comment (rileykarson) [reply CHANGELOG.md:8]: Ehh- both follow the style guide as-written.

File: CHANGELOG.md (hunk 1 above)
Comment (rileykarson) [reply CHANGELOG.md:12]:
```suggestion
```

**Pull request merged**